pkg/reconciler/v1alpha1/configuration/resources/build.go

```diff
@@ -48,6 +51,20 @@ func MakeBuild(config *v1alpha1.Configuration) *unstructured.Unstructured {
 
 		u = MustToUnstructured(b)
 	}
+	// After calling `As()` we can be sure that `.Raw` is populated.
+
+	// Compute the hash of the current build's spec.
+	hasher := sha256.New()
+	hasher.Write(config.Spec.Build.Raw)
```

Comment:
Should we normalize this so that changes in whitespace don't trigger a new build.

Reply:
I'm inclined to leave it as-is.  I'd wager the Kubernetes API Server already normalizes things and the raw bytes should come from there.